data/hai/unfettered side missions.txt

```diff
@@ -203,29 +139,55 @@ mission "Hai Legacy 1"
 					goto foolish
 			`	As you try to react, the Hai closest to you firmly grabs your arm and shouts through the noise. "Wait! If this ship wanted to kill you, <last>, you would already be dead."`
 			label wait
-			`	You can barely hear anything over the sound of the engines as the ship proceeds to land on the beach. Suddenly, all is calm. They turn to you, surprisingly calm. "This ship means you no harm. This is a mighty relic of the past, from the times our Empire used to control the stars: a Ladybug!`
+			`	You can barely hear anything over the sound of the engines as the ship proceeds to land on the beach. Suddenly, all is calm. They turn to you, the second one surprisingly calm but you can tell the first one was a bit tense, "This ship means you no harm," they say.`
```

Comment:
```suggestion
			`	You can barely hear anything over the sound of the engines as the ship proceeds to land on the beach. Suddenly, all is calm. They turn to you, both surprisingly at ease. "This ship means you no harm."`
```

Reply:
the idea is this was not planned, and one of the two is more apt than the other at managing her reaction to it

Reply:
```suggestion
			`	You can barely hear anything over the sound of the engines as the ship proceeds to land on the beach. Suddenly, all is calm. They turn to you, one more at ease than the other. "This ship means you no harm."`
```
like this?